jenkins/databricks/clusterutils.py

```diff
@@ -42,10 +42,10 @@ def generate_create_templ(sshKey, cluster_name, runtime, idle_timeout,
         templ['autotermination_minutes'] = idle_timeout
         templ['enable_elastic_disk'] = 'false'
         templ['enable_local_disk_encryption'] = 'false'
-        templ['node_type_id'] = worker_node_type
+        templ['node_type_id'] = driver_node_type
         templ['driver_node_type_id'] = driver_node_type
         templ['ssh_public_keys'] = [ sshKey ]
-        templ['num_workers'] = num_workers
+        templ['num_workers'] = 0
```

Comment:
These changes seem unrelated to the PR?

Reply:
Oops, let me droped it, seems I need to push -f to update the PR
Thanks @jlowe